src/AnnotatedCommand.php

```diff
@@ -225,8 +225,9 @@ protected function setCommandArguments($commandInfo)
      */
     protected function checkUsesInputInterface($params)
     {
+        /** @var \ReflectionParameter $firstParam */
         $firstParam = reset($params);
-        return $firstParam instanceof InputInterface;
+        return $firstParam->getClass() && $firstParam->getClass()->implementsInterface(InputInterface::class);
```

Comment:
The parameter here might be a boolean, which will cause `$firstParam->getClass()` to fail. `is_object($firstParam)` is probably a workable substitute here.